mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price, item) {
```

Comment:
The second parameter "item" is never used

Reply:
Ohh yeah, I forgot to delete that